de.js

```diff
@@ -0,0 +1,334 @@
+    about_my_doge: `Über My Doge`,
+    admin: `Admin`,
+    i_cloud: `iCloud`,
+    cloud: `Cloud`,
+    youve_backed_up_to: `Dein Backup ist in der`,
+    you_havent_created_a_cloud_backup: `Du hast noch kein Cloud-Backup erstellt`,
+    manage: `Verwalten`,
+    backup: `Backup`,
+    view_seed_phrase: `Seed-Phrase zeigen`,
+    pin: `PIN`,
+    back_up_to: `Backup per`,
+    back_up_your_wallet: `Erstelle ein Backup deiner Wallet`,
+    back_up_your_wallet_description: `Erstelle ein Backup deiner Wallet in der Cloud und schütze es mit einem Passwort. Du kannst deine Seed-Phrase auch aufschreiben oder in einen Passwort-Manager kopieren.`,
+    pin_description: `Wenn du deine PIN vergisst musst du deine Wallet aus einem Backup wiederherstellen.`,
+    change_pin: `PIN ändern`,
+    your_pin_is_required: `Deine PIN wird benötigt um Zahlungen und vertrauliche Aktionen durchzuführen`,
+    use_fingerprint: `Fingerabdruck/Gesichtserkennung benutzen`,
+    ask_every_time_i_open_the_app: `Jedes Mal fragen wenn ich die App öffne`,
+    this_will_remove_all_wallet: `Dadurch werden alle mit dieser Wallet in Zusammenhang stehenden Daten von deinem Gerät entfernt`,
+    delete_wallet: `Wallet löschen`,
+    sign_in_to_the_dogeverse: `Im Dogeverse anmelden`,
+    dogeverse_intro_description: `Sichere dir dein DogeTag, finde Freunde, erhalte Zahlungen über deinen Link und interagiere mit Inhalten im Dogeverse.`,
+    sign_in: `Anmelden`,
+    sign_in_description: `Deine Telefon-Nummer wird nur zum Anmelden benötigt und wird nicht öffentlich gemacht.`,
+    create_account_description: `Tritt dem Dogeverse bei und verbinde dich mit einem riesigen Netzwerk von Shibes.`,
+    send_verification: `Bestätigungscode senden`,
+    verify_your_phone_number: `Bestätige deine Telefon-Nummer`,
+    enter_the_code_sent_to: `Gib den Code ein. Er wurde gesendet an`,
+    resend_code: `Code erneut senden`,
+    wrong_number: `Falsche Nummer`,
+    confirm_code: `Code bestätigen`,
+    youve_entered_the_verse: `Du hast das Dogeverse betreten`,
+    youre_now_ready_to_explore_the_dogeverse: `Du bist nun bereit das Dogeverse zu erkunden!`,
+    lets_go: `Dann mal los!`,
+    edit_profile: `Profil bearbeiten`,
+    social_connect: `Verbundene Accounts`,
+    profile_privacy: `Privatsphäre`,
+    profile_is_public: `Profil ist öffentlich`,
+    show_wallet_address_and_tip_button: `Zeige Wallet-Adresse and Trinkgeld-Taste`,
+    preferences: `Einstellungen`,
+    confirm_x_tips: `Bestätigung für Trinkgelder per 𝕏`,
+    tips_received: `Erhaltene Trinkgelder`,
+    new_followers: `Neue Follower`,
+    new_memes: `Neue Memes`,
+    about_mydoge: `Über MyDoge`,
+    terms_of_use: `Nutzungsbedingungen`,
+    privacy_policy: `Datenschutzrichtlinie`,
+    share_mydoge: `MyDoge teilen`,
+    visit_our_website: `Besuche unsere Website`,
+    settings_and_support: `Einstellungen und Support`,
+    connect_twitch: `Twitch verbinden`,
+    connect_x: `𝕏 verbinden`,
```

Comment:
If change to @styleshifter's suggestion above then +'Mit ' here too ;)

```suggestion
    connect_x: `Mit 𝕏 verbinden`,
```